test/langtools/jdk/javadoc/doclet/testParamTaglet/pkg/C.java

```diff
@@ -38,4 +38,39 @@ public void regularParams(int param1, int param2) {}
      */
     public void nonMatchingParams(int param1, int param2) {}
 
+    /**
+     * Overriding method with missing/additional/duplicate param tags in non-declaration order.
+     *
+     * @param x does not exist
+     * @param b a boolean
+     * @param i an int
+     * @param x duplicate
+     * @param b another duplicate
+     */
+    public void unorderedParams(int i, double d, boolean b) {}
```

Comment:
Consider adding `@Override` to this method and to `nonMatchingParams`.

Reply:
Thanks for the feedback, Pavel. I added a commit that addresses all your suggestions.